neanderthal/src/main/kotlin/au/com/outware/neanderthal/data/repository/VariantSharedPreferencesRepository.kt

```diff
@@ -81,6 +90,17 @@ class VariantSharedPreferencesRepository(val klass: Class<out Any>,
         return variants
     }
 
+    override fun getDefaultVariants(): List<Variant> {
```

Comment:
using kotlin's functional programming methods, you could turn this whole function into
```
return sharedPreferences.getStringSet(DEFAULT_VARIANT_LIST, emptySet()).map { name ->
    Variant(name, gson.fromJson(sharedPreferences.getString(name, null), klass))
}.toMutableList()
```